<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect><x>0</x><y>0</y><width>518</width><height>727</height></rect>
        </property>
        <property name="minimumSize">
            <size><width>518</width><height>727</height></size>
        </property>
        <property name="maximumSize">
            <size><width>518</width><height>727</height></size>
        </property>
        <property name="windowTitle"><string>Satoshi++</string></property>
        <!-- Global button styling -->
        <property name="styleSheet">
            <string>
                QPushButton {
                background-color: #ffffff;
                color: #333333;
                border: 1px solid #cccccc;
                border-radius: 4px;
                padding: 6px 12px;
                font-family: Monospace;
                font-size: 14px;
                }
                QPushButton:hover { background-color: #f5f5f5; }
                QPushButton:pressed { background-color: #e6e6e6; }
                QPushButton:disabled {
                color: #999999;
                border-color: #dddddd;
                }
            </string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayoutRoot">
                <item>
                    <widget class="QStackedWidget" name="stackedWidget">

                        <!-- PAGE 0: Home -->
                        <widget class="QWidget" name="pageHome">
                            <layout class="QGridLayout" name="centralGridLayout">
                                <item row="0" column="0">
                                    <spacer name="spacerTop">
                                        <property name="orientation"><enum>Qt::Vertical</enum></property>
                                        <property name="sizeType"><enum>QSizePolicy::Expanding</enum></property>
                                    </spacer>
                                </item>
                                <item row="1" column="0">
                                    <layout class="QVBoxLayout" name="verticalLayoutHome">
                                        <item>
                                            <widget class="QLabel" name="label">
                                                <property name="font"><font><family>Monospace</family><pointsize>20</pointsize><bold>true</bold></font></property>
                                                <property name="text"><string>Welcome to Satoshi++</string></property>
                                                <property name="alignment"><set>Qt::AlignCenter</set></property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLabel" name="label_2">
                                                <property name="font"><font><family>Monospace</family></font></property>
                                                <property name="text"><string>Satoshi++ is a simple lightweight crypto wallet</string></property>
                                                <property name="alignment"><set>Qt::AlignCenter</set></property>
                                                <property name="wordWrap"><bool>true</bool></property>
                                            </widget>
                                        </item>
                                        <item>
                                            <layout class="QHBoxLayout" name="horizontalButtonLayout">
                                                <item><spacer name="spacerLeft"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                                <item><widget class="QPushButton" name="pushButton"><property name="text"><string>Create New</string></property></widget></item>
                                                <item><widget class="QPushButton" name="pushButton_2"><property name="text"><string>Open Existing</string></property></widget></item>
                                                <item><spacer name="spacerRight"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                            </layout>
                                        </item>
                                    </layout>
                                </item>
                                <item row="2" column="0">
                                    <spacer name="spacerBottomHome">
                                        <property name="orientation"><enum>Qt::Vertical</enum></property>
                                        <property name="sizeType"><enum>QSizePolicy::Expanding</enum></property>
                                    </spacer>
                                </item>
                            </layout>
                        </widget>

                        <!-- PAGE 1: Create Wallet -->
                        <widget class="QWidget" name="pageCreate">
                            <layout class="QVBoxLayout" name="verticalLayoutCreate">
                                <item>
                                    <layout class="QHBoxLayout" name="backLayoutCreate">
                                        <item><widget class="QPushButton" name="backButtonCreate"><property name="text"><string>← Back</string></property></widget></item>
                                        <item><spacer name="spacerBackCreate"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                    </layout>
                                </item>
                                <item><spacer name="spacerCreateTop"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                <item><widget class="QLabel" name="mnemonicLabel"><property name="font"><font><family>Monospace</family><pointsize>14</pointsize></font></property><property name="alignment"><set>Qt::AlignCenter</set></property><property name="wordWrap"><bool>true</bool></property></widget></item>
                                <item><widget class="QLabel" name="noteLabel"><property name="font"><font><family>Monospace</family><pointsize>12</pointsize><bold>true</bold></font></property><property name="text"><string>Keep this passphrase safe! It is your primary way to access your wallet</string></property><property name="alignment"><set>Qt::AlignCenter</set></property><property name="wordWrap"><bool>true</bool></property></widget></item>
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayoutCreateButtons">
                                        <item><spacer name="spacerCreateLeft"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                        <item><widget class="QPushButton" name="copyButton"><property name="text"><string>Copy</string></property></widget></item>
                                        <item><widget class="QPushButton" name="openWalletButton"><property name="text"><string>Open Wallet</string></property></widget></item>
                                        <item><spacer name="spacerCreateRight"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                    </layout>
                                </item>
                                <item><spacer name="spacerCreateBottom"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                            </layout>
                        </widget>

                        <!-- PAGE 2: Open Existing -->
                        <widget class="QWidget" name="pageExisting">
                            <layout class="QVBoxLayout" name="verticalLayoutExisting">
                                <item>
                                    <layout class="QHBoxLayout" name="backLayoutExisting">
                                        <item><widget class="QPushButton" name="backButtonExisting"><property name="text"><string>← Back</string></property></widget></item>
                                        <item><spacer name="spacerBackExisting"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                    </layout>
                                </item>
                                <item><spacer name="spacerExistingTop"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                <item><widget class="QLabel" name="existingNoteLabel"><property name="font"><font><family>Monospace</family><pointsize>14</pointsize></font></property><property name="text"><string>Paste your mnemonic</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
                                <item><widget class="QLineEdit" name="mnemonicInput"><property name="minimumSize"><size><width>300</width><height>30</height></size></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
                                <item><widget class="QPushButton" name="openExistingButton"><property name="text"><string>Open</string></property><property name="minimumSize"><size><width>100</width><height>30</height></size></property></widget></item>
                                <item><spacer name="spacerExistingBottom"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                            </layout>
                        </widget>

                        <!-- PAGE 3: Balance -->
                        <widget class="QWidget" name="pageBalance">
                            <layout class="QVBoxLayout" name="verticalLayoutBalance">
                                <item><spacer name="spacerBalanceTop"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                <item><widget class="QLabel" name="balanceInfoLabel"><property name="font"><font><family>Monospace</family><pointsize>18</pointsize><bold>true</bold></font></property><property name="text"><string>Balance: 0 ETH</string></property><property name="alignment"><set>Qt::AlignCenter</set></property><property name="wordWrap"><bool>true</bool></property></widget></item>
                                <item>
                                    <layout class="QHBoxLayout" name="layoutSendReceive">
                                        <item><spacer name="spacerSendLeft"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                        <item><widget class="QPushButton" name="sendButton"><property name="text"><string>Send</string></property></widget></item>
                                        <item><widget class="QPushButton" name="receiveButton"><property name="text"><string>Receive</string></property></widget></item>
                                        <item><spacer name="spacerSendRight"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                    </layout>
                                </item>
                                <item><widget class="QListWidget" name="coinList"><property name="minimumSize"><size><width>0</width><height>100</height></size></property><property name="frameShape"><enum>QFrame::NoFrame</enum></property><property name="focusPolicy"><enum>Qt::NoFocus</enum></property><property name="styleSheet"><string>
                                    QListWidget { background: transparent; border: none; font-family: Monospace; font-size:14px; }
                                    QListWidget::item { padding:12px; margin:4px 0; }
                                    QListWidget::item:selected { background:#555; color:#fff; border-radius:5px; border: none; }
                                </string></property></widget></item>
                                <item><spacer name="spacerBalanceBottom"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                <item>
                                    <layout class="QHBoxLayout" name="logoutLayout">
                                        <item><spacer name="spacerLogoutLeft"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                        <item><widget class="QPushButton" name="logoutButton"><property name="text"><string>Logout</string></property></widget></item>
                                        <item><spacer name="spacerLogoutRight"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                    </layout>
                                </item>
                            </layout>
                        </widget>

                        <!-- PAGE 4: Receive Address -->
                        <widget class="QWidget" name="pageReceive">
                            <layout class="QVBoxLayout" name="verticalLayoutReceive">
                                <item><spacer name="spacerReceiveTop"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                <item><widget class="QLabel" name="receiveTitleLabel"><property name="font"><font><family>Monospace</family><pointsize>18</pointsize><bold>true</bold></font></property><property name="text"><string>ETHEREUM Address</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
                                <item><widget class="QLabel" name="addressLabel"><property name="font"><font><family>Monospace</family><pointsize>14</pointsize></font></property><property name="text"><string>0x0000…</string></property><property name="alignment"><set>Qt::AlignCenter</set></property><property name="wordWrap"><bool>true</bool></property></widget></item>
                                <item>
                                    <layout class="QHBoxLayout" name="receiveButtonsLayout">
                                        <item><spacer name="spacerReceiveLeft"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                        <item><widget class="QPushButton" name="copyAddressButton"><property name="text"><string>Copy</string></property><property name="minimumSize"><size><width>100</width><height>30</height></size></property></widget></item>
                                        <item><spacer name="spacerReceiveRight"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                    </layout>
                                </item>
                                <item>
                                    <layout class="QHBoxLayout" name="backLayoutReceive">
                                        <item><spacer name="spacerBackReceiveLeft"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                        <item><widget class="QPushButton" name="backButtonReceive"><property name="text"><string>← Back</string></property><property name="minimumSize"><size><width>80</width><height>30</height></size></property></widget></item>
                                        <item><spacer name="spacerBackReceiveRight"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                    </layout>
                                </item>
                                <item><spacer name="spacerReceiveBottom"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                            </layout>
                        </widget>

                        <!-- PAGE 5: Send Transaction -->
                        <widget class="QWidget" name="pageSend">
                            <layout class="QVBoxLayout" name="verticalLayoutSend">
                                <item><spacer name="spacerSendTop"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                <item><widget class="QLabel" name="sendTitleLabel"><property name="font"><font><family>Monospace</family><pointsize>18</pointsize><bold>true</bold></font></property><property name="text"><string>Sending COIN</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
                                <item><widget class="QDoubleSpinBox" name="amountInput"><property name="minimumSize"><size><width>200</width><height>30</height></size></property><property name="decimals"><number>8</number></property><property name="singleStep"><double>0.0001</double></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
                                <item><widget class="QLineEdit" name="addressInput"><property name="placeholderText"><string>Recipient Address</string></property><property name="minimumSize"><size><width>300</width><height>30</height></size></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
                                <item>
                                    <layout class="QHBoxLayout" name="sendButtonsLayout">
                                        <item><spacer name="spacerSendButtonsLeft"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                        <item><widget class="QPushButton" name="sendTxButton"><property name="text"><string>Send</string></property></widget></item>
                                        <item><widget class="QPushButton" name="backButtonSend"><property name="text"><string>← Back</string></property></widget></item>
                                        <item><spacer name="spacerSendButtonsRight"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                                    </layout>
                                </item>
                                <item><spacer name="spacerSendBottom"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
                            </layout>
                        </widget>

                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar"/>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
